lib/rspec/rails/view_rendering.rb

```diff
@@ -81,7 +81,11 @@ def append_view_path(new_path)
       private
 
         def _path_decorator(path)
-          EmptyTemplateResolver.new(path)
+          if path.is_a?(::ActionView::Resolver)
+            path
```

Comment:
I must admit, I wasn’t entirely sure what `EmptyTemplateResolver` does. It seems to change the found templates to templates that look identical, but that always render an empty string when `call`ed.

I guess the same behavior can be applied if `path` is an instance of `ActionView::Resolver`. But we need to change the superclass of `EmptyTemplateResolver` to be `ActionView::Resolver`, right?